website/css/style.css

```diff
@@ -0,0 +1,563 @@
+/**
+reset5 © 2011 opensource.736cs.com MIT
+https://code.google.com/archive/p/reset5/
```

Comment:
TODO added. Adding a CSS/JS compiler would help, but I don't think we are that desperate yet.